docs/src/deployment/examples.md

```diff
@@ -143,11 +143,11 @@ while also providing the most redundancy and security for your dedicated game se
 ## What Next?
 
 * Have a look at the [example configurations](./examples.md) for configuration and usage examples.
```

Comment:
With this rename, this now points to itself, we should either remove this line, replace it with something else or put back in a page for listing examples?